{{ cookiecutter.package_name }}/docs/conf.py

```diff
@@ -69,15 +70,14 @@
 # This does not *have* to match the package name, but typically does
 project = setup_cfg['name']
 author = setup_cfg['author']
-copyright = '{0}, {1}'.format(
-    datetime.datetime.now().year, setup_cfg['author'])
+copyright = '{0}, {1}'.format(datetime.datetime.now().year, author)
```

Comment:
we dropped <python3.6, so why not f strings? :trollface:

Reply:
I feel like f-string is a package-wide clean-up PR? 😬